static/app/utils/dates.tsx

```diff
@@ -34,6 +36,19 @@ export function getUtcDateString(dateObj: moment.MomentInput): string {
   return moment.utc(dateObj).format(moment.HTML5_FMT.DATETIME_LOCAL_SECONDS);
 }
 
+/**
+ * Given a date field from a table row, return a timestamp
+ * given: '2024-04-01T20:15:18+00:00'
+ * returns: 1712002518
+ */
+export function getTimeStampFromTableDateField(date: keyof TableDataRow): number {
+  if (typeof date === 'number') {
+    return date;
+  }
+
+  return new Date(date).getTime() / 1000;
```

Comment:
Not sure if this is possible from the data, but getTime can return NaN for invalid dates, you might want to check for that

const time = new Date(date).getTime();
if(isNaN(time)) 
  // throw or return some fallback value
else 
  return time/1000